src/main/java/com/toofifty/easygiantsfoundry/EasyGiantsFoundryPlugin.java

```diff
@@ -175,4 +212,29 @@ public void onScriptPostFired(ScriptPostFired event)
 			mouldHelper.selectBest(event.getScriptId());
 		}
 	}
+
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
```

Comment:
Remove this method, since the boolean variables should also be removed and config can be used directly.